### ISSUE TITLE
Add autocomplete attributes to form elements

### ISSUE THREAD

**Issue by witoszekdev** [saleor/react-storefront#374]
Add autocomplete attributes to form elements so that browsers and password maangers (e.g. 1Password) can easily enter correct data in the forms automatically.

Shipping and billing address should be differentiated by using `shipping` and `billing` values respectively.

Also, to consider: billing inputs should be rendered but hidden, so that extensions can fill those values automatically, and we can show them if values for shipping and billing aren't the same

## Usefull links
HTML spec: https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#attr-fe-autocomplete

**Comment by mmiszy** [comment]
Fixed in https://github.com/saleor/react-storefront/pull/477
